rasa/core/policies/memoization.py

```diff
@@ -239,20 +251,52 @@ def predict_action_probabilities(
         return self._prediction(result)
 
     def _metadata(self) -> Dict[Text, Any]:
-        return {
-            "priority": self.priority,
-            "max_history": self.max_history,
-            "lookup": self.lookup,
-        }
+        return {"lookup": self.lookup}
 
     @classmethod
     def _metadata_filename(cls) -> Text:
         return "memorized_turns.json"
 
+    @classmethod
+    def load(
+        cls,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+        **kwargs: Any,
+    ) -> "MemoizationPolicyGraphComponent":
```

Comment:
please don't use forward references (you can remove them by importing `from __future__ import annotations`)
```suggestion
    ) -> MemoizationPolicyGraphComponent:
```